CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add main thread ID to profiling payload (#1918)
 - Add App Hangs tracking (#1906)
+- Track timezone changes as breadcrumbs (#1930)
```

Comment:
`m`: We just released a new version. This needs to move up to a new `## Unreleased` section.